app/managers.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite3")
+        self.table_name = "actors"
+
+    def all(self) -> list[Actor]:
+        cursor = self._connection.execute("SELECT * FROM actors")

```

Comment:
It's better to follow SQL style guidelines even while writing raw SQL in Python. Therefore I'd suggest to start FROM clause from new line.